### PR TITLE
[12.0][MIG] l10n_nl_account_tax_unece

### PR DESCRIPTION
**Pull request by astirpe**

### CODE REVIEW

File: l10n_nl_location_nuts/tests/test_l10n_nl_location_nuts.py [hunk 1]
```diff
@@ -10,11 +10,15 @@ class TestNlLocationNuts(TransactionCase):
     def setUp(self):
```
Comment (NL66278) [inline l10n_nl_location_nuts/tests/test_l10n_nl_location_nuts.py:10]: The changes to this module should not be here I think.

File: l10n_nl_account_tax_unece/post_install.py [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import api, SUPERUSER_ID
+
+
+def set_unece_on_taxes(cr, _):
+    with api.Environment.manage():
+        env = api.Environment(cr, SUPERUSER_ID, {})
+        for company in env['res.company'].search([]):
+            if company.country_id and company.country_id != env.ref('base.nl'):
```
Comment (NL66278) [inline l10n_nl_account_tax_unece/post_install.py]: Would it not be more safe to have this only work for companies that have country_id explicitly set to the Netherlands?

Comment (astirpe) [reply l10n_nl_account_tax_unece/post_install.py]: Right, thanks!

File: l10n_nl_account_tax_unece/post_install.py [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import api, SUPERUSER_ID
+
+
+def set_unece_on_taxes(cr, _):
+    with api.Environment.manage():
+        env = api.Environment(cr, SUPERUSER_ID, {})
+        for company in env['res.company'].search([
+            ("country_id", "=", env.ref("base.nl").id),
```
Comment (NL66278) [inline l10n_nl_account_tax_unece/post_install.py]: country_id on res.company is a non-stored field, leading to the travis errors. However searching on partner_id,country_id should solve this. Strangely enough when I use an odoo shell to directly search on country_id in res.company, like in your code, I do get the expected result as well... Even after checking that country_id is indeed not stored.

**Review by NL66278 — APPROVED** [review]
:+1: LGTM

**Pull request merged**